libcontainerd/local/local_windows.go

```diff
@@ -1374,3 +1371,40 @@ func (c *client) reapProcess(ctr *container, p *process) int {
 
 	return exitCode
 }
+
+// reapContainer shuts down the container and releases associated resources. It returns
+// the error to be logged in the eventInfo sent back to the monitor.
+func (c *client) reapContainer(ctr *container, p *process, exitCode int, exitedAt time.Time, eventErr error, logger *logrus.Entry) (int, error) {
+	// Update container status
+	ctr.Lock()
+	ctr.status = libcontainerdtypes.StatusStopped
+	ctr.exitedAt = exitedAt
+	ctr.exitCode = uint32(exitCode)
+	close(ctr.waitCh)
+
+	if err := c.shutdownContainer(ctr); err != nil {
+		exitCode = -1
+		logger.WithError(err).Warn("failed to shutdown container")
+		thisErr := fmt.Errorf("failed to shutdown container: %s", err)
+		if eventErr != nil {
+			eventErr = fmt.Errorf("%s: %s", eventErr, thisErr)
+		} else {
+			eventErr = thisErr
+		}
+	} else {
+		logger.Debug("completed container shutdown")
+	}
+	ctr.Unlock()
+
+	if err := ctr.hcsContainer.Close(); err != nil {
+		exitCode = -1
+		logger.WithError(err).Error("failed to clean hcs container resources")
+		thisErr := fmt.Errorf("failed to terminate container: %s", err)
+		if eventErr != nil {
+			eventErr = fmt.Errorf("%s: %s", eventErr, thisErr)
```

Comment:
errors.Wrap

Reply:
> Are the package moves due to platform dependent code imports?

@cpuguy83 No, it's because we are now in this interim stage where previously, "remote" was only called from linux (due to conditional compile) and "local" was only called from Windows (again due to conditional compile). We're temporarily in a hybrid state where Windows can use both local (the default) and remote (the experimental option). Ultimately local will be removed, remote will be the only thing in libcontainerd, things will be more normal-looking, but it's a little way off yet.

Reply:
> errors.Wrap

Fixed in next push